modules/applications/pages/ui-builder/how-to-interact-with-tasks.adoc

```diff
@@ -37,69 +60,91 @@ For example:
 For more detailed information, refer to the {openApiUrl}/latest/#tag/UserTask/operation/getContextByUserTaskId[context by userTask id api documentation].
 
 
-=== User task execution
-The user task execution API executes a task identified by its id. 
+=== Business data
+The business data is useful to find business data with pagination params and filters by invoking a `Named Query`. It allows to call either default or custom business data queries.
 
 For example:
 
-    - POST | `/bonita/API/bpm/userTask/[taskId]/execution`
+    - GET | `/API/bdm/businessData/[businessObject]?q=[query]&p=0&c=10&f=[filterString]=[filterParam]`
+    - GET | `/API/bdm/businessData/com.company.model.Claim?c=1000&p=0&q=find`
 
-The task contract values have to be provided in the query's body:
+`q=[query]` is a mandatory parameter. Queries are used to get information about data objects stored in the database. A set of xref:data:define-and-deploy-the-bdm.adoc#queries[default queries], `find` and `find_by`, is defined automatically for each object in the BDM. You can also create custom queries. 
```

Comment:
:warning: Is better to put a real anchor instead of using the section title.

Reply:
See for more details: https://bonitasoft.github.io/bonita-documentation-theme/links.html#_anchors_to_navigate_inside_the_same_document

Reply:
Archor fixed 👍